.devcontainer/devcontainer.json

```diff
@@ -14,12 +14,22 @@
   },
   "runArgs": [
     "--volume=/tmp/.X11-unix:/tmp/.X11-unix",
-    "--volume=${localWorkspaceFolder}/.devcontainer/.Xauthority:/root/.Xauthority",
-    "--volume=${localEnv:HOME}/.comma:/root/.comma",
+    "--volume=${localWorkspaceFolder}/.devcontainer/.Xauthority:/home/batman/.Xauthority",
+    "--volume=${localEnv:HOME}/.comma:/home/batman/.comma",
     "--volume=/tmp/comma_download_cache:/tmp/comma_download_cache",
     "--volume=/tmp/devcontainer_scons_cache:/tmp/scons_cache",
     "--shm-size=1G"
   ],
+  "features": {
+    "ghcr.io/devcontainers/features/common-utils:2": {
```

Comment:
this features thing is pretty cool, perhaps we should use it instead of stuff like this https://github.com/commaai/openpilot/pull/29967

Reply:
I think they aren't as useful for one-off features like this, unless we wanted to have a devcontainer for the submodules too. They require creating docker artifacts, and need a whole CI setup. Doesn't look like they support local features.

https://github.com/devcontainers/feature-starter